lta/rest_server.py

```diff
@@ -392,28 +437,46 @@ class TransferRequestSingleHandler(BaseLTAHandler):
     @lta_auth(roles=['admin', 'user', 'system'])
     async def get(self, request_id: str) -> None:
         """Handle GET /TransferRequests/{uuid}."""
-        if request_id not in self.db['TransferRequests']:
+        # if request_id not in self.db['TransferRequests']:
+        #     raise tornado.web.HTTPError(404, reason="not found")
+        # self.write(self.db['TransferRequests'][request_id])
+        ret = await self.db.TransferRequests.find_one({'uuid': request_id}, REMOVE_ID)
+        if not ret:
             raise tornado.web.HTTPError(404, reason="not found")
-        self.write(self.db['TransferRequests'][request_id])
+        else:
+            self.write(ret)
 
     @lta_auth(roles=['admin', 'user', 'system'])
     async def patch(self, request_id: str) -> None:
         """Handle PATCH /TransferRequests/{uuid}."""
-        if request_id not in self.db['TransferRequests']:
-            raise tornado.web.HTTPError(404, reason="not found")
         req = json_decode(self.request.body)
         if 'uuid' in req and req['uuid'] != request_id:
             raise tornado.web.HTTPError(400, reason="bad request")
-        self.db['TransferRequests'][request_id].update(req)
-        self.write({})
+        # self.db['TransferRequests'][request_id].update(req)
+        # self.write({})
+        sbtr = self.db.TransferRequests
+        query = {"uuid": request_id}
+        update = {"$set": req}
+        ret = await sbtr.find_one_and_update(query,
+                                             update,
+                                             REMOVE_ID,
```

Comment:
Be explicit about what parameter `REMOVE_ID` is.
```suggestion
                                             projection=REMOVE_ID,
```